src/models/game-media.model.ts

```diff
@@ -0,0 +1,11 @@
+export interface Media {
+  audios: any[];
+  videos: any[];
+  images: [
+    {
+      url: string;
+      format: string;
+      type: string;
+    }
+  ];
```

Comment:
This means `images` is a singleton containing an object with these three properties. I have not yet run the code so I am assuming this is a mistake (feel free to correct me if it is not!). You may be looking for:

```ts
images: Array<{
  url: string;
  format: string;
  type: string;
}>
```

Even better if we can also strongly type `format` and `type` -- I wonder if there is a known quantity for these props that is more specific than `string`.

Reply:
You are correct, it's an array of objects with those properties. I'll make that change soon.